cmd/di.go

```diff
@@ -19,6 +19,7 @@ package cmd
 
 import (
 	"encoding/json"
+	"github.com/mysteriumnetwork/node/port"
```

Comment:
i would go with 
```suggestion
	"github.com/mysteriumnetwork/node/core/pools/port"
```

Reply:
Moved to `node/core/port`. Pool is an implementation detail, so I wanted to avoid that.